src/components/common/copy/copy.tsx

```diff
@@ -0,0 +1,35 @@
+import * as React from 'react';
+import { useState } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCopy, faCheck } from '@fortawesome/pro-regular-svg-icons';
+import copy from 'copy-to-clipboard';
+
+import * as styles from './copy.module.scss';
+
+type Props = {
+  text: string;
+};
+
+const Copy: React.FC<Props> = ({ text }) => {
+  const [copied, setCopied] = useState<boolean>(false);
+  const handleCopy = (): void => {
+    copy(text);
+    setCopied(true);
+    setTimeout(() => setCopied(false), 1500);
```

Comment:
```suggestion
```

Reply:
I didn't want to leave any comment, but this is too much. It gives better UX.

Reply:
oh didn't notice that it's for ui, let's leave it and just move 1500 to constants at the top of file